basic-algorithm-scripting/confirm-the-ending.js

```diff
@@ -0,0 +1,9 @@
+function confirmEnding(str, target) {
+  if (str.substr(-target.length) === target) {
```

Comment:
altere sua logica para utilizar o metodo .substring pois o metodo .substr esta defasado e deve ser descontinuado.
Com o .substring a logica do numero negativo nao vai funcionar portanto vc vai ter que pensar em uma alternativa.

Reply:
Again!
if your boolean expression return the result you want just return this directly dont use an if structure in these cases.
Example...
if (true) {
  return true
} else {
 return false
}

This expression doesnt make sense because your boolean expression already return the right boolean. So just return the expression and dont build an if structure.